sage-ios/SAGE/SAGE/ErrorView.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  ErrorView.swift
+//  SAGE
+//
+//  Created by Sameera Vemulapalli on 10/24/15.
+//  Copyright © 2015 Cal Blueprint. All rights reserved.
+//
+
+import UIKit
+
+class ErrorView: UIView {
+
+    var message: UILabel = UILabel()
+    var background: UIView = UIView()
+    
+    required override init(frame: CGRect) {
```

Comment:
take this out, or make it call the convenience init(height, string) with some default values for those parameters